share/instruments/instruments.xml

```diff
@@ -1066,10 +1066,13 @@
                   <shortName>Hph.</shortName>
                   <description>Heckelphone</description>
                   <musicXMLid>wind.reed.heckelphone</musicXMLid>
-                  <clef>G8vb</clef>
+                  <clef>G</clef>
```

Comment:
`<clef>` should be `<transposingClef>`, and `<concertClef>` should appear immediately afterwards. [See piccolo](https://github.com/musescore/MuseScore/blob/84d4d73dd744b30cedcf2b9e23b1db1708a383c4/share/instruments/instruments.xml#L88-L89) for example.

Reply:
My bad, I'll change that and update the branch